.github/workflows/build.yml

```diff
@@ -0,0 +1,58 @@
+name: Build the Python app
+run-name: ${{ github.actor }} - ${{ github.ref_name }}
+
+on:
+    push:
+        branches: ["main", "develop"]
+    pull_request:
+        branches: ["main"]
+
+env:
+    PYTHON_VERSION: 3.13
```

Comment:
The Python version `3.13` does not exist. Please update it to a valid version, such as `3.11`, which is the latest stable release.